CONTRIBUTING.md

```diff
@@ -1,4 +0,0 @@
-Contributing
```

Comment:
Perhaps keep this file for now but instead mention it's deprecated and point to readme.md for full details? (don't want to duplicate info)